addons/cni-cilium/poddisruptionbudgets.yaml

```diff
@@ -0,0 +1,45 @@
+{{ $version := semver .Config.Versions.Kubernetes }}
+{{- if ge $version.Minor 21 }}
+apiVersion: policy/v1
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
+kind: PodDisruptionBudget
+metadata:
+  name: cilium-operator
+  namespace: kube-system
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      name: cilium-operator
+---
+{{- if ge $version.Minor 21 }}
+apiVersion: policy/v1
+{{- else }}
+apiVersion: policy/v1beta1
+{{- end }}
+kind: PodDisruptionBudget
+metadata:
+  name: hubble-relay
+  namespace: kube-system
+spec:
+  minAvailable: 1
+  selector:
+    matchLabels:
+      k8s-app: hubble-relay
```

Comment:
Hubble is optional, so we should probably create PDB only if it's deployed.

Reply:
Not that such "dangling" PDB objects would harm anyone, but anyway, surrounded them with the condition.